tests/test_integration.py

```diff
@@ -50,7 +51,10 @@ def test_build_wheel(distribution: str) -> None:
     "distribution", os.listdir(os.path.join(TYPESHED, THIRD_PARTY_NAMESPACE))
 )
 def test_version_increment(distribution: str) -> None:
-    get_version.determine_stub_version(read_metadata(TYPESHED, distribution))
+    try:
+        get_version.determine_stub_version(read_metadata(TYPESHED, distribution))
+    except AlreadyUploadedError:
+        pass
```

Comment:
Is it worth asserting that the last part of the version is equal to today's date if it raised AlreadyUploadedError?